src/parser.ts

```diff
@@ -1,48 +1,50 @@
 import { Natspec, NatspecDefinition } from './types/natspec.t';
 import { NodeToProcess } from './types/solc-typed-ast.t';
 
-export function parseNodeNatspec(node: NodeToProcess): Natspec {
-  if (!node.documentation) {
-    return { tags: [], params: [], returns: [] };
-  }
+export class Parser {
+  parseNodeNatspec(node: NodeToProcess): Natspec {
```

Comment:
does it make sense to have a class for one single function? aren't we just adding unnecessary complexity and extra steps here?